### PR TITLE
Fixed code to use db.changes

### PR DESCRIPTION
**Pull request by emlaver**
- Simplified python code to submit `doc` to the Db2 node
- Added additional doc fields to attributes table
- Updated streams flow file
- Added note about Cloudant account is the username for most cases
- Updated running state streams flow image

### CODE REVIEW

File: streams-designer/README.md [hunk 1]
```diff
@@ -96,32 +94,33 @@ The first node we'll create is a Python node for loading documents from Cloudant
...
+               output_doc['wiki_page'] = doc['wiki_page']
+           if 'min_weight' in doc:
+               output_doc['min_weight'] = doc['min_weight']
+           if 'max_weight' in doc:
+               output_doc['max_weight'] = doc['max_weight']
+           if 'class' in doc:
+               output_doc['class'] = doc['class']
+           if 'diet' in doc:
+               output_doc['diet'] = doc['diet']
+           # Submit a tuple in each iteration:
```
Comment (ricellis) [inline streams-designer/README.md]: I'm confused, isn't `output_doc` a dict rather than a tuple?

File: streams-designer/README.md [hunk 2]
```diff
@@ -96,32 +94,33 @@ The first node we'll create is a Python node for loading documents from Cloudant
...
+           submit(output_doc)
```
Comment (ricellis) [inline streams-designer/README.md]: Remind me why we can't just `submit(doc)`?
Do we need to ensure the fields only match the ones we are expecting? If so could we just do:
```python
output_doc = { prop: doc[prop] for prop in ['_id','_rev','wiki_page','min_weight','max_weight','class','diet'] if prop in doc }
```

**Review by ricellis — COMMENTED** [review]
A couple of questions

File: streams-designer/README.md (hunk 2 above)
Comment (emlaver) [reply streams-designer/README.md]: @ricellis UPDATED: It didn't take much effort to add all the doc fields and fix up the tutorial.  The python code is now simplified to just passing `doc` in d193710.

File: streams-designer/README.md (hunk 1 above)
Comment (emlaver) [reply streams-designer/README.md]: Changed to `Submit a Cloudant doc in each iteration` in d193710.

**Review by ricellis — APPROVED** [review]
+1

**Pull request merged**